src/main/java/sevensmurfs/rehub/model/message/request/validator/PasswordResetValidator.java

```diff
@@ -0,0 +1,10 @@
+package sevensmurfs.rehub.model.message.request.validator;
+
+import jakarta.validation.groups.Default;
+
+public interface PasswordResetValidator {
+
+    interface getPasswordResetLink extends Default {}
```

Comment:
No needed but these are actually inner classes. Follow naming so it starts with capital at least. e.g ResetLink and SaveNew